pager.c

```diff
@@ -3067,7 +3075,7 @@ int mutt_pager(const char *banner, const char *fname, PagerFlags flags, struct P
       case OP_PURGE_MESSAGE:
       case OP_DELETE:
         CHECK_MODE(IsEmail(extra));
-        CHECK_READONLY;
+        if (!assert_mailbox_writable(Contex2->mailbox)) break;
```

Comment:
You asked about these `break`s...
The look OK, but `clang-format` will wrap them.